dask/array/optimization.py

```diff
@@ -127,21 +156,21 @@ def optimize_slices(dsk):
     fancy_ind_types = (list, np.ndarray)
     dsk = dsk.copy()
     for k, v in dsk.items():
-        if type(v) is tuple and v[0] in GETTERS and len(v) in (3, 5):
+        if (get := _is_getter_task(v)) and len(v) in (3, 5):
```

Comment:
I think this would be the first walrus operator usage in the dask org

Reply:
I checked for it last night, and I believe so :)

We'll see if it sticks, given your below suggestion